src/main/java/com/github/maxopoly/WurstCivTools/ConfigParser.java

```diff
@@ -71,6 +72,22 @@ public void parseTools(ConfigurationSection config) {
 						+ showNonRunning + ", showUpgrading:" + showUpgrading
 						+ ", cooldown:" + cd);
 				break;
+			case "PLAYERHOOK":
+				if (current.getBoolean("enabled",false) == false){
+					plugin.info("PlayerHook disabled, skipping...");
+					continue;
+				}
+				boolean prevent_swords = current.getBoolean("prevent_use_with_swords", false);
+				boolean prevent_axes = current.getBoolean("prevent_use_with_axes", false);
+				boolean prevent_bows = current.getBoolean("prevent_use_with_bows", false);
```

Comment:
Can you add an example config to the current config. I guess the current one is used as prod config, so just comment it out I guess?